pages/signup_in_page.py

```diff
@@ -9,4 +9,28 @@ def __init__(self, driver):
     input_name = '//input[@data-qa="signup-name"]'
     input_email = '//input[@data-qa="signup-email"]'
     signup_button = '//button[@data-qa="signup-button"]'
-    signup_page_text = '//*[text()="Enter Account Information"]'
\ No newline at end of file
+    signup_page_text = '//*[text()="Enter Account Information"]'
+    
+    input_email_login = '//input[@data-qa="login-email"]'
+    password = '//input[@data-qa="login-password"]'
+    login_button = '//button[@data-qa="login-button"]'
+
+    # error_message  = '//*[text()="Your email or password is incorrect!"]'
+    # error_message = '//p[contains(text(), "Your email or password is incorrect")]'
+    # error_message = '//form[@action="/login"]//p[contains(normalize-space(.), "Your email or password is incorrect!")]'
+    # error_message = '//p[contains(normalize-space(.), "Your email or password is incorrect")]'
```

Comment:
```suggestion
    error_message = "//div[@class='login-form']//p[text()='Your email or password is incorrect!']"
```
перевіряла Тетяна, зуб дє що воно процює